docs/README.md

```diff
@@ -60,6 +60,7 @@
     * [some(selector)](/docs/api/ShallowWrapper/some.md)
     * [someWhere(predicate)](/docs/api/ShallowWrapper/someWhere.md)
     * [state([key])](/docs/api/ShallowWrapper/state.md)
+    * [tap()](/docs/api/ShallowWrapper/tap.md)
```

Comment:
oh and let's call the argument by the same name it is called in the doc for `tap` to be consistent with the other methods, so `intercepter` instead of `fn`